PhysicsTools/TensorFlow/bin/test_tfgraph.cc

```diff
@@ -0,0 +1,154 @@
+/*
+ * Test of the TensorFlow graph interface.
+ *
+ * Graph (from bin/data/simplegraph, created by test/creategraph.py):
+ *   Input:
+ *     - op = "input", op index = 0, shape = (batch, 10)
+ *   Output:
+ *     - op name = "output", op index = 0, shape = (batch, 1)
+ *
+ * Usage:
+ *   > test_tfgraph
+ *
+ * Author:
+ *   Marcel Rieger
+ */
+
+#include <stdio.h>
+#include <iostream>
+#include <string>
+#include <vector>
+
+#include <PhysicsTools/TensorFlow/interface/Tensor.h>
+#include <PhysicsTools/TensorFlow/interface/Graph.h>
+
+void test(bool success, const std::string& msg)
+{
+    if (success)
+    {
+        std::cout << "    " << msg << std::endl;
+    }
+    else
+    {
+        throw std::runtime_error("test failed: " + msg);
+    }
+}
+
+int main()
+{
+    std::cout << std::endl << "test tf::Graph" << std::endl;
+
+    // get the directory that contains the simple SavedModel
+    std::string cmsswBase = std::string(getenv("CMSSW_BASE"));
+    std::string graphDir = cmsswBase + "/src/PhysicsTools/TensorFlow/bin/data/simplegraph";
```

Comment:
it would be better to write to the test directory instead of the source code:
```${LOCAL_TEST_DIR}``` is set to some derivative of ```$CMSSW_BASE/test/``` when the test runs with ```scram b runtests```